src/olympia/pages/templates/pages/shield_study_9.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Get Synced</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_9/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Get Synced</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/get-synced-v1-0/?src=shield-study-9"><button class="button primary large">Get It Now</button></a>
```

Comment:
Just noting that this still ends up in a 404 because the add-on wasn't fixed / updated yet. It should work and looks correct to me but we can't verify just yet.